src/frontend/screens/Invites/InviteReceived.tsx

```diff
@@ -44,6 +51,11 @@ export const InviteReceived = ({
   const {formatMessage} = useIntl();
   const inviteId = route.params.inviteId;
   const {data: invite} = useSingleInvite({inviteId});
+  const translatedRole =
+    invite?.roleName === 'Coordinator'
```

Comment:
optional chain isn't needed here
```suggestion
    invite.roleName === 'Coordinator'
```